app/src/main/java/by/academy/jc/ht/shorokhov/ht13/Person.java

```diff
@@ -0,0 +1,114 @@
+package by.academy.jc.ht.shorokhov.ht13;
+
+import com.google.common.collect.Sets;
+
+import java.util.*;
+
+public class Person {
+    String name;
+    String surname;
+    String mySurname = "шорохов";
+    int age;
+
+
+    public Person(String name, String surname, int age) {
+        this.age = age;
+        this.surname = surname;
+        this.name = name;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+
+    public String getSurname() {
+        return surname;
+    }
+
+
+    public int getAge() {
+        return age;
+    }
+
+
+    public String printInfo(){
+        return (getName() + " " + getSurname() + ", " + getAge() + " y.o.");
+    }
+
+
+    public boolean isCurrentSurname(){
```

Comment:
`Person` is POJO, and `isCurrentSurname` is predicate, business function. It should be placed in separate predicate class or in lambda expression